exercise_0/characterCounter/CharacterCounter.cpp

```diff
@@ -0,0 +1,61 @@
+//
+// Created by emyrmak on 7/3/2019.
+//
+#include <iostream>
+#include "CharacterCounter.h"
+
+CharacterCounter::CharacterCounter() {
+    vowelsCount = 0;
+    consonantsCount = 0;
+    digitsCount = 0;
+    othersCount = 0;
```

Comment:
Use member initialization list for such stuff
```cpp
CharacterCounter::CharacterCounter() : vovelsCount{ 0 }, consonantsCount{ 0 } etc... {}
```
Or even in class members initializers.

Reply:
Did not know about it